common/resource/resource.go

```diff
@@ -94,6 +94,7 @@ type (
 		// persistence clients
 
 		GetMetadataManager() persistence.MetadataManager
+		GetClusterMetadataManager() persistence.ClusterMetadataManager
```

Comment:
Why not just to add it to version checker?

Reply:
This was left out before, just fixing it here.